src/core/types.ts

```diff
@@ -210,21 +210,39 @@ export type InternalRefetchQueriesInclude =
   | InternalRefetchQueryDescriptor[]
   | RefetchQueriesIncludeShorthand;
 
-// Used by ApolloClient["refetchQueries"]
-// TODO Improve documentation comments for this public type.
+/**
+ * Options object for the `client.refetchQueries` method.
+ * @public
+ */
 export interface RefetchQueriesOptions<TCache extends ApolloCache, TResult> {
+  /**
+   * Optional function that updates cached fields to trigger refetches of queries that include those fields.
+   */
   updateCache?: (cache: TCache) => void;
-  // The client.refetchQueries method discourages passing QueryOptions, by
-  // restricting the public type of options.include to exclude QueryOptions as
-  // an available array element type (see InternalRefetchQueriesInclude for a
-  // version of RefetchQueriesInclude that allows legacy QueryOptions objects).
+
+  /**
+   * Optional array specifying queries to refetch. Each element can be either a query's string name or a `DocumentNode` object.
+   *
+   * Pass `"active"` (or `"all"`) as a shorthand to refetch all (active) queries.
```

Comment:
```suggestion
   * Pass `"active"` as a shorthand to refetch all active queries, or `"all"` to refetch all active and inactive queries.
```

Might be good to be a bit more explicit here.